### PR TITLE
CSPWFRT: Enrollment confirmation email

### PR DESCRIPTION
**Pull request by islemaster**
Fix up the enrollment confirmation email for the new CSP Workshop for Returning Teachers workshop type. ([spec](https://docs.google.com/document/d/1u1Vfm5DnsBpwUrV2XLp9XMcF2R3SU7b1EAtlBBB6wEU/edit#))

image

## Testing story

Tested using rails mailer previews.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by islemaster** [comment]
Note: There are a few decisions in here that are still waiting on confirmation from Megan.

File: dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml [hunk 1]
```diff
@@ -46,7 +46,7 @@
     = render partial: 'csp_pre_course_reading'
 
 %p
-  If you have any accessibility needs or food allergies, please let the workshop organizer know so they can make appropriate accommodation plans and answer any of your questions. You can reach them at
+  If you have any accessibility needs or food allergies, please contact the workshop organizer to discuss appropriate accommodation plans and to answer any of your questions. You can reach them at
   = mail_to(@workshop.organizer.email) + '.'
```
Comment (islemaster) [inline dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml:50]: It's intentional that this language change applies to all of our workshop types, not just the new one.

**Comment by bencodeorg** [comment]
LGTM -- tangent based on the mailer preview, did you set the fake workshop you're working with to last two days? Do we have to do anything to limit workshop length to a day? I know we have a work item to limit the hours on certificates, but I think that's separate.

**Review by bencodeorg — APPROVED** [review]

**Comment by islemaster** [comment]
I haven't tried that out yet, no.  I think in general we want to lift limits on how many days/sessions a workshop can go, since there may be more variety in the setup of virtual workshops.

**Pull request merged**